packages/scene-composer/src/models/SceneModels.ts

```diff
@@ -151,6 +152,13 @@ export namespace Component {
     selector: string | number;
   }
 
+  export interface Animation extends IComponent {
+    selector?: number;
+    uri: string;
+    currentAnimations: string[];
+    allAnimations: string[];
```

Comment:
I might miss something, but where is this set and used? this interface should match with the `IAnimationComponent` in `src/interfaces/components` file